inference-engine/tests/functional/inference_engine/transformations/eliminate_split_test.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <queue>
+#include <ngraph/op/parameter.hpp>
+#include <ngraph/opsets/opset8.hpp>
+#include <transformations/common_optimizations/nop_elimination.hpp>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+
+using namespace testing;
+
+TEST_F(TransformationTestsF, EliminateSplit) {
+    {
+        auto input = std::make_shared<ngraph::opset8::Parameter>(ngraph::element::f32, ngraph::PartialShape::dynamic());
+        auto mul_constant = ngraph::opset8::Constant::create(ngraph::element::f32, ngraph::Shape{1}, {89.2});
+        auto mul = std::make_shared<ngraph::opset8::Multiply>(input, mul_constant);
+        auto axis_const = ngraph::opset8::Constant::create(ngraph::element::i64, ngraph::Shape{}, {2});
+        auto split = std::make_shared<ngraph::opset8::Split>(mul, axis_const, 1);
+        auto res = std::make_shared<ngraph::opset8::Result>(split);
+        function = std::make_shared<ngraph::Function>(ngraph::NodeVector{res}, ngraph::ParameterVector{input});
+
+        manager.register_pass<ngraph::pass::EliminateSplit>();
+    }
+    {
+        auto input = std::make_shared<ngraph::opset8::Parameter>(ngraph::element::f32, ngraph::PartialShape::dynamic());
+        auto mul_constant = ngraph::opset8::Constant::create(ngraph::element::f32, ngraph::Shape{1}, {89.2});
+        auto mul = std::make_shared<ngraph::opset8::Multiply>(input, mul_constant);
+        auto res = std::make_shared<ngraph::opset8::Result>(mul);
+        function_ref = std::make_shared<ngraph::Function>(ngraph::NodeVector{res}, ngraph::ParameterVector{input});
+    }
+}
+
+TEST_F(TransformationTestsF, EliminateSplitNegative) {
+    {
+        auto input = std::make_shared<ngraph::opset8::Parameter>(ngraph::element::f32, ngraph::PartialShape::dynamic());
+        auto mul_constant = ngraph::opset8::Constant::create(ngraph::element::f32, ngraph::Shape{1}, {89.2});
+        auto mul = std::make_shared<ngraph::opset8::Multiply>(input, mul_constant);
+        auto axis_const = ngraph::opset8::Constant::create(ngraph::element::i64, ngraph::Shape{}, {2});
+        auto split = std::make_shared<ngraph::opset8::Split>(mul, axis_const, 3);
+        auto res1 = std::make_shared<ngraph::opset8::Result>(split->output(0));
+        auto res2 = std::make_shared<ngraph::opset8::Result>(split->output(1));
+        auto res3 = std::make_shared<ngraph::opset8::Result>(split->output(2));
+        function = std::make_shared<ngraph::Function>(ngraph::NodeVector{res1, res2, res3}, ngraph::ParameterVector{input});
+
+        manager.register_pass<ngraph::pass::EliminateSplit>();
+    }
+
+    {
+        auto input = std::make_shared<ngraph::opset8::Parameter>(ngraph::element::f32, ngraph::PartialShape::dynamic());
```

Comment:
If you expect that transformation doesn't change Function, you can skip reference graph creation so the original function will be used for comparison.

Reply:
ok. Done